src/models/Models.jl

```diff
@@ -12,6 +12,7 @@ using MLDataPattern
 using ProgressMeter: Progress, next!
 
 include("utils/utils.jl")
+include("utils/2stage.jl")
```

Comment:
I would probably like it more if it was two_stage instead of 2stage, files beginning with numbers don't look nice.

Reply:
done